### PR TITLE
Add feeder cacheing

### PR DESCRIPTION
**Pull request by josephmckinsey**
Solves #75

### CODE REVIEW

**Review by AadilLatif — DISMISSED** [review]

**Comment by AadilLatif** [comment]
@josephmckinsey tests seem to be failing

FileNotFoundError: [Errno 2] Failed to open local file 'outputs_build/voltage_real.feather'. Detail: [errno 2] No such file or directory

File: LocalFeeder/FeederSimulator.py [hunk 1]
```diff
@@ -129,8 +129,9 @@ def __init__(self, config: FeederConfig):
 
         self.tap_setting = config.tap_setting
```
[the remark this answers is not in the document]
Comment (josephmckinsey) [reply LocalFeeder/FeederSimulator.py:131]: Issue #75

**Review by tarekelgindy — APPROVED** [review]
So this now uses the default unit of seconds for the run. This works correctly without specifying the "s" at the end when it's run right?

**Comment by josephmckinsey** [comment]
> So this now uses the default unit of seconds for the run. This works correctly without specifying the "s" at the end when it's run right?

From the opendss manual:
> You may specify the stepsize in seconds, minutes, or hours by appending ‘s’, ‘m’, or ‘h’ to the size value. If omitted, ‘s’ is assumed. Example: “set stepsize=15m” is the same as “set stepsize=900.” Do not leave a space between the value and the character

**Pull request merged**